deploy/operator/utils.sh

```diff
@@ -9,6 +9,18 @@ function wait_for_crd() {
     wait_for_condition "crd/${crd}" "Established" "60s" "${namespace}"
 }
 
+function remote_agents() {
+	oc get agent -n ${REMOTE_AGENT_NAMESPACE} --no-headers -l "agent-install.openshift.io/bmh in ( ${comma_sep_host_names} )"
+}
```

Comment:
the implicit dependency on `REMOTE_AGENT_NAMESPACE` and `comma_sep_host_names` is a bit hard to understand, I suggest doing it as common in this file:
```suggestion
function remote_agents() {
	namespace="$1"
	hostnames="$2"
	oc get agent -n ${namespace} --no-headers -l "agent-install.openshift.io/bmh in ( ${hostnames} )"
}
```

Reply:
Done